docs/style_guide.adoc

```diff
@@ -96,79 +94,24 @@ g_environment: someval
 ----
 
 === Ansible Role Variables
-Ansible role variables are defined as variables contained in (or passed into) a role.
-
 '''
-[[Role-variables-MUST-have-a-prefix-of-atleast-3-characters-See.below.for.specific.naming.rules]]
+[[Ansible-role-variable-prefix]]
 [cols="2v,v"]
 |===
-| <<Role-variables-MUST-have-a-prefix-of-atleast-3-characters-See.below.for.specific.naming.rules, Rule>>
-| Role variables MUST have a prefix of at least 3 characters. See below for specific naming rules.
+| <<Ansible-role-variable-prefix, Rule>>
+| Role variables MUST have a prefix of `r_<role_name>_`
 |===
 
-==== Role with 3 (or more) words in the name
+.Definition
+Ansible variables contained in or passed into a role.
 
-Take the first letter of each of the words.
-
-.3 word example:
-* Role name: made_up_role
-* Prefix: mur
-[source]
-----
-mur_var1: value_one
-----
+.Purpose
+Name variables to avoid name collision between roles or playbooks.
```

Comment:
**Other requirements**:

If a role `roleB` depends on `roleA`, all role variables from `roleA` that affects value of role variables in `roleB` must have their equivalents defined in  `roleB`.
E.g.
* `roleA` defines among others `r_roleA_runtime` and `r_roleA_conf_dir` variables
* `roleB` defines `r_roleB_service_file` which depends on `r_roleA_runtime` variable
* then, `roleB` must defined `r_roleB_runtime` variable